app/views/patron_requests/_scan_options.html.erb

```diff
@@ -23,7 +23,7 @@
       <ul data-itemselector-target="scanItem" class="list-unstyled d-flex flex-wrap gap-2 bg-white m-0 me-3"></ul>
     <% else %>
       <div class="text-cardinal">
-        Earliest expected delivery: <%= f.object.scan_earliest['display_date'] %>
+        Earliest expected delivery: <%= f.object.scan_earliest['display_date'] %> - <%= queue_length_display(f.object.items_in_location.first) %>
```

Comment:
Does this need to vary based on the actual selected item? If there's 2 copies and one has a a request and one doesn't, do we still want to show the waitlist info?

Reply:
This else is if there is only one item on the page.